packages/asset-swapper/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "4.7.0",
```

Comment:
⚠️  this should probably be a major version bump since we change the return type of a public function